test/e2e/installplan_e2e_test.go

```diff
@@ -3192,14 +3192,20 @@ var _ = Describe("Install Plan", func() {
 
 		It("should clear clear up the condition in the InstallPlan status that contains an error message when a valid OperatorGroup is created", func() {
 
-			// first check that a condition with a message exists
-			fetchedInstallPlan, err := fetchInstallPlanWithNamespace(GinkgoT(), crc, installPlanName, ns.GetName(), buildInstallPlanPhaseCheckFunc(operatorsv1alpha1.InstallPlanPhaseInstalling))
-			Expect(err).NotTo(HaveOccurred())
-			Expect(fetchedInstallPlan).NotTo(BeNil())
```

Comment:
Based on [this](https://github.com/operator-framework/operator-lifecycle-manager/blob/dd804774acab78de3d7165d9461789e3c66920ba/test/e2e/installplan_e2e_test.go#L4099) return statement within the `fetchInstallPlanWithNamespace` function, it is possible that the `fetchedInstallPlan` will be nil if the `Get` call from the client ever returns `nil, nil`.

I can't think of a time where this could happen, but we should probably safeguard against it.

Reply:
Yes, that's right.  I'll add nil check for fetchedInstallPaln.  Thanks!